src/pint/utils.py

```diff
@@ -1120,6 +1139,28 @@ def pferrs(porf, porferr, pdorfd=None, pdorfderr=None):
 
     Calculate the period or frequency errors and
     the pdot or fdot errors from the opposite one.
+
+    Parameters
+    ----------
+    porf : :class:`astropy.units.Quantity`
+        pulsar period (or frequency)
```

Comment:
This is horrible. Can we please have an `fperrs` that maybe calls this one to do the calculation but both have sensibly named arguments and docstrings? `porf` -> `p` in one and `f` in the other for example?

Reply:
I didn't write this function, just documented it.  So I didn't want to change the API.

Reply:
It is a direct port from psr_utils in presto. It would be fine to propose a new version, but that would be a different pull request.

Reply:
Haha!  This is copied from PRESTO.  It's been in there for *18 years!* and I've used it a lot.  Kills two birds with one stone!   Minimizes copied code!   Super clever!  :-)

Reply:
Zero copied code:
```python
def fperrs(f, ferr, ...):
    return pferrs(f, ferr, ...)
```
but the names and docstring could be intelligible.